examples/grid-runner-example.py

```diff
@@ -0,0 +1,48 @@
+import sem
+import os
+
+
+# Define campaign parameters
+############################
+
+script = 'wifi-multi-tos'
+ns_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'ns-3')
+campaign_dir = "/nfsd/signet/ns/magrin/grid-runner-testing"
+
+# Create campaign
+#################
+
+campaign = sem.CampaignManager.new(ns_path, script, campaign_dir,
+                                   runner='GridRunner',
+                                   overwrite=False)
+
+print(campaign)
+
+# Run simulations
+#################
+
+# Parameter space
+#################
+nWifi_values = [1]
+distance_values = [1]
+simulationTime_values = [10]
+useRts_values = ['false', 'true']
+mcs_values = list(range(8))
+channelWidth_values = ['20']
+useShortGuardInterval_values = ['false', 'true']
+runs = 2
```

Comment:
If we have time, I think it could be a good idea to compare the performance (e.g., total running time) between cluster and single machine runners.

Reply:
Sure! We can definitely do a comparison between:
1. Sequential simulations
2. Parallel multiprocessor simulations on a single machine
3. Cluster simulations